packages/canary-web-components/src/components/ic-pagination-bar/test/a11y/ic-pagination-bar.test.a11y.ts

```diff
@@ -9,6 +9,15 @@ describe("pagination bar component", () => {
       "<ic-pagination-bar total-items='100'></ic-pagination-bar>"
     );
     checkShadowElementRendersCorrectly(el);
-    expect(await axe(el)).toHaveNoViolations();
+    expect(
+      await axe(el, {
+        // Tested pagination in Cypress and this error doesn't occur, not sure what's causing it in this test
```

Comment:
Is this comment still suppose to be here ?

Reply:
Yeah it's just to make people aware when they do the Cypress migration 😊

Reply:
Sure thing! Just wanted to check :)